recipes/hunspell/build.sh

```diff
@@ -0,0 +1,17 @@
+autoreconf -vfi
+./configure --prefix=$PREFIX --with-readine --with-ui
+make
+make check
+make install
+mv $PREFIX/bin/hunspell $PREFIX/bin/.hunspell
+
+# We have to make a wrapper to add a spelling dictionary path to hunspell,
+# since none of the default ones are relative to the binary (i.e., can be
+# installed as a conda package)
+cat <<EOF > $PREFIX/bin/hunspell
+#!/bin/sh
+export DICPATH='/opt/anaconda1anaconda2anaconda3/hunspell_dictionaries'
```

Comment:
Shouldn't `/opt/anaconda1anaconda2anaconda3/` be `$PREFIX`?

Reply:
Actually, this is sort of a bad hack.  It can't be $PREFIX, because that would evaluate to the build time location, where as this needs to be the install location.

Reply:
`$PREFIX` is the install location right?

Reply:
Ahh, actually $PREFIX works